try {
    let a = 10;
    console.log(Program);
} catch (err) {
    console.log("Error");
    console.log(Program);

} finally {
    console.log("Execute Every Time")
}